mmv1/third_party/terraform/services/storagetransfer/resource_storage_transfer_job_test.go

```diff
@@ -381,6 +432,99 @@ resource "google_storage_transfer_job" "transfer_job" {
 `, project, dataSourceBucketName, project, dataSinkBucketName, project, transferJobDescription, project)
 }
 
+func testAccStorageTransferJob_eventStream(project string, dataSourceBucketName string, dataSinkBucketName string, transferJobDescription string, eventStreamStart string, eventStreamEnd string) string {
+	return fmt.Sprintf(`
```

Comment:
To make the changes I mentioned in other review comments you'll need to add a lot more arguments into `fmt.Sprintf` and you'll need to make sure they're in the right number and there's enough `%s`'s in the string. To avoid that, search the codebase for use of `acctest.Nprintf()`. It'll let you format a string in similar ways but with less repetition or arguments!

Reply:
This change is optional, but I think it'll make your life easier